includes/admin/class-wc-subscriptions-admin.php

```diff
@@ -136,7 +136,6 @@ public static function init() {
 
 		// Change text for when order items cannot be edited
 		wcs_add_woocommerce_dependent_action( 'woocommerce_admin_order_totals_after_total', array( __CLASS__, 'maybe_attach_gettext_callback' ), '4.0.0', '>' );
```

Comment:
If we no longer need to support 4.0.0 or earlier, can we also simplify this line?

```suggestion
		add_action( 'woocommerce_admin_order_totals_after_total', array( __CLASS__, 'maybe_attach_gettext_callback' ) );
```

It feels like we no longer need the added indirection that `wcs_add_woocommerce_dependent_action()` and `WCS_Dependent_Hook_Manager` provide in this case.

Reply:
Fixed with dff1a5df844fb6fc4caf45c1865a86f2c05a946b